src/enclave/rpcsessions.h

```diff
@@ -62,6 +62,11 @@ namespace enclave
       certs.push_back(std::move(the_cert));
     }
 
+    void add(std::shared_ptr<tls::Cert> cert)
```

Comment:
What is this used for?

Reply:
It's a leftover from the cert changes, will take it out.